test/notaryUtils.test.js

```diff
@@ -50,6 +50,50 @@ describe('notaryUtils.setupCase', function () {
     assert.equal(caseID, notary + '/cases/' + pathParts[pathParts.length - 1])
     caseNock.done()
   })
+
+  it('creates a case ID and then returns it on 200', function * () {
```

Comment:
`it('returns the case ID on 200 -- passed in caseID')` might be clearer, otherwise LGTM